modules/EvseManager/Charger.cpp

```diff
@@ -524,26 +543,22 @@ void Charger::runStateMachine() {
                     break;
                 }
 
-                if (new_in_state) {
+                if (initialize_state) {
                     signalEvent(types::evse_manager::SessionEventEnum::ChargingPausedEV);
-                    r_bsp->call_allow_power_on(true);
-                    // make sure we are enabling PWM
-                    if (hlc_use_5percent_current_session) {
-                        update_pwm_now(PWM_5_PERCENT);
-                    } else {
-                        update_pwm_now(ampereToDutyCycle(getMaxCurrent()));
-                    }
+                    // bsp->allow_power_on(true, types::evse_board_support::Reason::FullPowerCharging);
```

Comment:
Is there a reason these calls are commented out?

Reply:
removed the commented out commands